### PR TITLE
Behave introduced

### PR DESCRIPTION
**Pull request by w1stler**
Initial setup + folders structure introduced.

### CODE REVIEW

**Comment by w1stler** [comment]
Please review: @yellowee , @danutaj23 , @monikakrawczak

File: pah_behave/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Go to the directory with PAH website, eg.:
+
+1. ```cd codeforpoznan/pah_fm/pah_behave```
+
+# Running behave tests
+
+Make sure, that virtual environment is active, so the prompt will look like:
+
+1. ```(pah_behave) ~/codeforpoznan/pah-fm/pah_ccbehave```
```
Comment (yellowee) [inline pah_behave/README.md]: It is an example, but should it have (pah_behave) ~/codeforpoznan/pah-fm/pah_**cc**behave` ?

File: pah_behave/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+1. ```behave pah_ccbehave/features/```
```
Comment (yellowee) [inline pah_behave/README.md]: ditto ;)

File: pah_behave/README.md (hunk 1 above)
Comment (w1stler) [reply pah_behave/README.md]: done

File: pah_behave/README.md (hunk 2 above)
Comment (w1stler) [reply pah_behave/README.md]: done

**Pull request merged**